modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,162 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="replacing-a-bare-metal-control-plane-node_{context}"]
+= Replacing a bare-metal control plane node
+
+Use the following procedure to replace an installer-provisioned {product-title} cluster.
+
+[IMPORTANT]
+====
```

Comment:
It is very important to note that the old bmh, machine and node objects need to be deleted before doing anything else.

Reply:
```
$ oc delete node kni1-vmaster-0
$ oc delete bmh -n openshift-machine-api vmaster-0
$ oc delete machine -n openshift-machine-api kni1-master-0
```

Reply:
Updated as suggested.